src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/alert-title-template.mdx

```diff
@@ -1,67 +1,87 @@
 ---
-title: Title template for NRQL Alert Conditions
+title: Create a title for a NRQL alert condition
+tags:
+  - Alerts and applied intelligence
+  - Alerts
+  - Alert conditions
+metaDescription: 'Give your alert condition a unique title to easily identify related incidents.'
+redirects:
+  - /docs/alerts/new-relic-alerts/defining-conditions/alert-condition-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/alert-custom-violation-descriptions/
+freshnessValidatedDate: never
 ---
 
-For NRQL alert conditions, you can define a title template.
+import alertsAddingAlertDetailstoanAlertCondition from 'images/alerts_screenshot-crop_adding-alert-details-to-an-alert-condition.webp'
 
-## What is a title template?
+An alert condition is the foundation for incident creation. Assigning a meaningful title to your alert condition ensures you can quickly pinpoint the source and specific parameters that triggered an incident. Whether you're using our guided mode or crafting a condition from a chart or policy, leverage our title template in the final step to customize incident titles for easy identification.
```

Comment:
I much prefer what I wrote here with `What is a title template`. It's clear, succinct, and explains exactly what a title template is. If I'm a user and I read what you've written here, I'm still not sure what a title template is. 

Furthermore, it's not really accurate.
> Assigning a meaningful title to your alert condition
This would be better as `Creating a meaningful title template for your alert creation.